web/static/css/_person-cards.scss

```diff
@@ -1,27 +1,46 @@
 .person-cards {
   display: flex;
   flex-wrap: wrap;
+  margin-bottom: $base-spacing;
 }
 
 .person-card {
   @include margin(null 2% 2% null);
-  @include padding($large-spacing $base-spacing null);
   background-color: $white;
+  border: $base-border;
   border-radius: $base-border-radius;
-  box-shadow: 0 3px 12px rgba($dark-gray, 0.1);
+  box-shadow: $base-box-shadow;;
   display: flex;
   flex-direction: column;
-  width: 32%;
+  width: 100%;
 
-  &:nth-child(3n) {
-    margin-right: 0;
+  @include media($small-screen-only) {
+    width: 48%;
+
+    &:nth-child(2n) {
```

Comment:
I think there's a little bigger mental leap between `3n` and `even`, vs `3n` and `2n`.

If I were just alternating a background color on rows, then I'd go with `even`.